include/kos/thread.h

```diff
@@ -690,26 +743,12 @@ int thd_pslist(int (*pf)(const char *fmt, ...));
 */
 int thd_pslist_queue(int (*pf)(const char *fmt, ...));
 
-/** \brief   Initialize the threading system.
-
-    This is normally done for you by default when KOS starts. This will also
-    initialize all the various synchronization primitives.
-
-    \retval -1              If threads are already initialized.
-    \retval 0               On success.
-
-    \sa thd_shutdown
-*/
+/** \cond Handled internally by KOS */
```

Comment:
Yeah, that's what `\cond` and `\endcond` which I've added do... but I guess what you'd like to see is both the conditional stuff plus the existing doxygen comments...

Reply:
Well, I don't know how it would work well. It makes sense not to clutter up doxygen with stuff that end users should neither know about nor have visibility into (not part of the 'public' API as much as we have such a thing). At the same time we'd want to have our own internal nice documentation for it to easily grok things.

For this though it's super pointless since the documentation is just `thd_init inits threads 0 is good -1 is bad`, but would be something to keep in mind for any internals that can actually benefit from docs.